src/pages/dashboard/OperatorHbarChart/OperatorHbarChart.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import './operatorsHbarChart.scss'
+
+const colorsByCompannies: { [index: string]: string } = {
+  'אגד תעבורה': 'rgb(47, 146, 80)',
```

Comment:
consider using scss var names for colors

Reply:
after using the scss var names - should I control the color using a different classname for every company?
I don't think it's scalable... in the future, we could add a static config file to load the colors from

Reply:
not scalable, but we don't really expect scale here.. anyway yeah, define it in some one place this is not the specific component that uses it now, especially if expect to use it across the system

Reply:
maybe we can store the color at `open-bus-stride-client/agencies/agencyList.ts`?

Reply:
sure